cypress/integration/pages/onDemandTV/index.js

```diff
@@ -13,9 +13,13 @@ Object.keys(config)
     const paths = getPaths(serviceId, pageType);
     paths.forEach(currentPath => {
       describe(`${pageType} - ${currentPath}`, () => {
-        before(() => {
+        beforeEach(() => {
           Cypress.env('currentPath', currentPath);
-          visitPage(currentPath, pageType);
+          const newPath =
+            Cypress.env('APP_ENV') === 'test'
+              ? `${currentPath}?renderer_env=live`
+              : currentPath;
```

Comment:
Could you use `overrideRendererOnTest()` here too?

Reply:
Done :)